### ISSUE TITLE
Great job on your work.

### ISSUE THREAD

**Issue by GanjinZero** [iiis-ai/IterativeQuestionComposing#1]
IQC shows significant improvement on MATH! I consider there may be a problem putting MATH output on GitHub directly which may cause future pre-training leakage.

**Comment by haoxiongliu** [comment]
Thanks for your attention and advice! We have removed the MATH test set and the output of models now.
